packages/react-i18n/README.md

```diff
@@ -42,17 +42,15 @@ export default function App() {
 
 ### Internationalized components
 
-Components must connect to the i18n context in order to get access to the many internationalization utilities this library provides. You can use the `withI18n` decorator to add an `i18n` prop to your component:
+Components must connect to the i18n context in order to get access to the many internationalization utilities this library provides. You can use the `useI18n` hook to access `i18n` in your component:
```

Comment:
Probably want to document `ShareTranslations` too but we can wait on that for the "combined" final PR